CHANGELOG.md

```diff
@@ -15,6 +15,8 @@ Canonical reference for changes, improvements, and bugfixes for Boundary.
 * metrics: Adds accepted connections and closed connections counters to keep track
   downstream connections for worker and controller servers.
   ([PR](https://github.com/hashicorp/boundary/pull/2668))
+* ui: Upgrade admin UI to Ember to 4.4 ([PR](https://github.com/hashicorp/boundary-ui/pull/1523)).
```

Comment:
Not sure what we really want to say here, but "admin ui to Ember to 4.4" sounds redundant to me.
How about?
- "Upgrade Admin UI to Ember 4.4" or "Upgrade Admin UI from Ember 3.28.x to Ember 4.4". Not sure how descriptive we want to be.

Reply:
Yeah that does read odd. I am not sure what I was thinking when I typed it 😆 
Thanks for the suggestions. I will update now.